wdl-lint/RULES.md

```diff
@@ -26,7 +26,7 @@ be out of sync with released packages.
 | `ImportSort`                     | Clarity, Style                | Ensures that imports are sorted lexicographically.                                                |
 | `ImportWhitespace`               | Clarity, Style, Spacing       | Ensures that there is no extraneous whitespace between or within imports.                         |
 | `InconsistentNewlines`           | Clarity, Style                | Ensures that newlines are used consistently within the file.                                      |
-| `InputSorting`                   | Style                         | Ensures that input declarations are sorted                                                        |
+| `InputSorting`                   | Style, Clarity, Sorting       | Ensures that input declarations are sorted                                                        |
```

Comment:
@a-frantz - Out-of-scope for this PR, but we should consider sorting these. It looks like many of them are already sorted alphabetically, but we could clean that up and adopt a policy to sort the entries. I didn't look at the in-code lists, but I'm not sure alphabetical is as important there.

Reply:
O I actually thought this was already supposed to be sorted alphabetically. IMO it's not that far out of scope. I think I'd rather get this sorted in this PR than spinning up a whole new one just for that.

For the in-code lists I think we've just been appending, which is fine to me. That's not worth the hassle to sort IMO.